src/App.css

```diff
@@ -0,0 +1,44 @@
+.App {
```

Comment:
***Code Quality: Critical***
css converstaions follow the use of lowercase wit dashes like this
```.app
.app-log
```
etc...